lib/js/components/Chip/Chip.vue

```diff
@@ -120,12 +126,20 @@ $pill-colors: (
 			'background': $color-default-background,
 		),
 	),
+	'invertedHex': (
```

Comment:
the name is a bit misleading because it is not inverted hex

Reply:
Any proposal?

Reply:
just inverted?